jira7/jira_connector.bal

```diff
@@ -19,834 +19,759 @@
 import ballerina/http;
 import ballerina/log;
 
-# Represents Jira Client Connector Object.
-public type JiraConnector object {
+# Object to initialize the connection with Jira.
+#
+# + jiraClient - The HTTP Client
+public type JiraConnector client object {
```

Comment:
Shall we remove this `.bal` file after moving the implementation into `jira_endpoint.bal`.
Please refer https://github.com/wso2-ballerina/module-twitter/pull/74